scriptoocloud/pom.xml

```diff
@@ -14,11 +14,11 @@
         <thymeleaf.version>3.0.11.RELEASE</thymeleaf.version>
 
         <admin.password>$2a$04$oQo44vqcDIFRoYKiAXoNheurzkwX9dcNmowvTX/hsWuBMwijqn44i</admin.password>
-        
-        <db.driver>com.mysql.jdbc.Driver</db.driver>
-        <db.database.url>jdbc-url-to-db</db.database.url>
-        <db.user>db-user</db.user>
-        <db.password>db-password</db.password>
+         
+        <db.driver>com.mysql.cj.jdbc.Driver</db.driver>
+        <db.database.url>jdbc:mysql://localhost:3306/test</db.database.url>
+        <db.user>root</db.user>
+        <db.password>root</db.password>
```

Comment:
try avoiding pushing your local configurations to GitHub. You can use Maven's settings.xml file instead.